### PR TITLE
engine(metrics): add server monitor to engine

### PR DESCRIPTION
**Pull request by CharlesCheung96**
### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tiflow/issues/6985

### What is changed and how it works?
1. Add server monitor to engine.
2. Add a new variable `instance`.
3. Optimize the existing panel measurement units.

#### Tests 

 - Manual test (add detailed scripts or steps below)
https://snapshots.raintank.io/dashboard/snapshot/U80Ntir3piC16oCHkp5joIR89oWUrVxp?orgId=2

### Release note 

```release-note
Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

If you don't think this PR needs a release note then fill it with `None`.
```

### CODE REVIEW

File: metrics/grafana/tiflow.json [hunk 1]
```diff
@@ -9,13 +9,231 @@
...
+      "model": {
+        "aliasColors": {},
+        "bars": false,
+        "dashLength": 10,
+        "dashes": false,
+        "datasource": {
+          "type": "prometheus",
+          "uid": "${DS_PROMETHEUS}"
+        },
+        "description": "Uptime of TiCDC and TiKV",
```
Comment (amyangfei) [inline metrics/grafana/tiflow.json:26]: Should update the description, ditto for other places with `TiCDC`

File: metrics/grafana/tiflow.json [hunk 2]
```diff
@@ -9,13 +9,231 @@
...
+          {
+            "datasource": {
+              "type": "prometheus",
+              "uid": "${DS_PROMETHEUS}"
+            },
+            "editorMode": "code",
+            "expr": "(time() - process_start_time_seconds{k8s_cluster=\"$k8s_cluster\", instance=~\"$instance\"})",
+            "format": "time_series",
+            "intervalFactor": 1,
+            "legendFormat": "Tiflow-{{instance}}",
```
Comment (amyangfei) [inline metrics/grafana/tiflow.json:74]: Is it possible to distinguish whether the role of instance is tiflow master or tiflow executor.

Comment (CharlesCheung96) [reply metrics/grafana/tiflow.json:74]: Currently, components can only be distinguished by specifying `job_name` at promethues config file.

**Review by maxshuang — APPROVED** [review]

**Review by amyangfei — APPROVED** [review]

**Comment by CharlesCheung96** [comment]
/run-integration-test tidb=f5487e3f2596cdd63ae9840606c7fff3fb539e03

**Comment by CharlesCheung96** [comment]
/run-integration-test tidb=f5487e3f2596cdd63ae9840606c7fff3fb539e03

**Pull request merged**